src/segment_git.go

```diff
@@ -3,19 +3,32 @@ package main
 import (
 	"bytes"
 	"fmt"
+	"net/url"
 	"strconv"
 	"strings"
 )
 
+var (
+	// Map for git service providers
+	gitProvidersMap map[string]string = map[string]string{
+		"github.com":       "%s/tree/%s",
+		"azure.com":        "%s?version=GB%s",
+		"visualstudio.com": "%s?version=GB%s",
+		"gitlab.com":       "%s/-/tree/%s",
```

Comment:
we should also add bitbucket, and that said, add support for these remotes too while we're at it. Maybe that got me thinking, this can be part of that configuration. The remote url match is hardcoded but can be rewritten and easily extended this way. Maybe that's out of scope for this change, but still a good idea.

Reply:
resolved by moving git segment to another pull request